seg_lapa/config_parse/callbacks_conf.py

```diff
@@ -16,8 +17,8 @@ class CallbacksConf(ABC):
     def get_callbacks_dict(self):
         return {}
 
-    def get_callbacks_list(self):
-        callback_dict = self.get_callbacks_dict()
+    def get_callbacks_list(self, log_dir):
+        callback_dict = self.get_callbacks_dict(log_dir)
```

Comment:
```suggestion
    def get_callbacks_list(self, *args):
        callback_dict = self.get_callbacks_dict(*args)
```
Since this is an abstract class, it's better to indicate that some optional positional arguments may be passed by `*args`. For named arguments, we'd use `**kwargs`.

This way, if the child classes want, they may accept additional positional arguments, but are not forced to.